dist/google-keep-card.js

```diff
@@ -83,33 +82,29 @@ class GoogleKeepCard extends LitElement {
                     margin: 0px;
                     padding: 0px;
                 }
-                
                 #googleKeepCard .smallTitleMargin {
                     padding-bottom: 4px;
-                }
-                    
+                }   
                 #googleKeepCard .card  {
                     margin: 16px 0px;
                     padding: 12px 16px;
                     position: relative;
                     overflow-wrap: break-word;
+                    opacity: ${this._alpha};
```

Comment:
I see :-) I wanted to keep only the background color to be influenced by the alpha and keep the text always readable! I will make sure to keep in mind that the whole card should be influence by this setting :-)

Reply:
Ok, that is my mistake